pkg/webreg/webreg.go

```diff
@@ -611,37 +611,74 @@ Example:
+</p>
+<ul>
+  <li>the <code>:src</code> tag, published as <code>:repo-scripts</code> containing the latest version of the repository to allow for executing helper scripts</li>
+  <li>the <code>:test-bin</code> tag, published as <code>:repo-tests</code> containing built test binaries to allow for running the repository's tests</li>
+  <li>the <code>:component</code> tag, published as <code>:component</code> containing the component itself to allow for deployments and installations in end-to-end scenarios</li>
+</ul>
+
+<code>ci-operator</code> config:
+{{ yamlSyntax (index . "imagePromotionConfig") }}
+
+<h4 id="image-consumption">Consuming an Image</h3>
+<p>
+Once a repository is publishing an image for reuse by others, downstream users can
+configure <code>ci-operator</code> to use that image in tests by including it as a
+<code>base_image</code> or as part of the <code>tag_specification</code>. In general,
+images will be available as part of the <code>tag_specification</code> and explicitly
+including them as a <code>base_image</code> will only be necessary if the promoting
+repository is exposing them to a non-standard <code>ImageStream</code>. The following
+<code>ci-operator</code> imports a number of images:
+</p>
+<ul>
+  <li>the <code>:custom-scripts</code> tag, published as <code>myregistry.com/project/custom-scripts:latest</code></li>
+  <li>the <code>:component</code> and <code>:repo-{scripts|tests}</code> tags, by virtue of them being published under <code>ocp/4.4</code> and brought in with the <code>tag_specification</code></li>
+</ul>
+
+<code>ci-operator</code> config:
+{{ yamlSyntax (index . "imageConsumptionConfig") }}
+
+<p>
+Once the image has been configured to be an input for the repository's tests in the
+<code>ci-operator</code> configuration, either explicitly as a <code>base_image</code>
+or implicitly as part of the <code>tag_specification</code>, it can be used in tests
+in one of two ways. A registry reference step can be written to execute the shared tests
+in any <code>ci-operator</code> config, or a literal test step can be added just to one
+repository's configuration to run the shared tests. Two examples follow which add an
+execution of shared end-to-end tests using these two approaches. Both examples assume
+that we have the <code>ipi</code> workflow available to use.
+</p>
 
-<h4>Method 1: Adding a new reference:</h4>
+<h5>Adding a Reusable Test Step:</h4>
 
 <p>
-Make directory for repo: <code>ci-operator/step-registry/org/repo/setup/project</code>
+Make directory for the test step in the registry: <code>ci-operator/step-registry/org/repo/tests/e2e</code>
 </p>
 
-Make reference: <code>org-repo-tests-custom-ref.yaml</code>
+Declare a reference: <code>org-repo-tests-e2e-ref.yaml</code>
 {{ yamlSyntax (index . "imageExampleRef") }}
 
-Make command: <code>org-repo-tests-integration-commands.sh</code>
+Populate a command file: <code>org-repo-tests-e2e-commands.sh</code>
```

Comment:
where does this file live?
how does it get injected into the test image (stable:repo-tests in this case?)